dev-python/flask-paginate/flask-paginate-0.5.5.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( pypy3 python3_{6,7} )
+
+inherit distutils-r1
+
+MY_COMMIT="510ad833106134711868653fb597bf75ea8ac34f"
+
+DESCRIPTION="Pagination support for flask"
+HOMEPAGE="https://flask-paginate.readthedocs.io"
+# https://github.com/lixxu/flask-paginate/issues/68
+SRC_URI="https://github.com/lixxu/flask-paginate/archive/${MY_COMMIT}.tar.gz -> ${P}.tar.gz"
```

Comment:
`${MY_COMMIT}` is usually used when upstream doesn't use tags to mark releases, I checked now and they started using tags so for new version you can use `${PV}`  here and drop `${MY_COMMIT}`